ddev/ddev-redis
# Add removal_action to remove settings.ddev.redis.php on add-on removal

## The Issue

In DDEV v1.22.0+ there is a `removal_actions` section of the install.yaml, specifically for things like what this add-on does, adding settings config into sites/default.

## How This PR Solves The Issue

Do the cleanup of sites/default/settings.ddev.redis.php on removal

## Manual Testing Instructions

On a drupal site:

`ddev get https://github.com/rfay/ddev-redis/tarball/20230720_post_removal_action`
Verify that sites/default/settings.ddev.redis.php exists
`ddev get --remove redis`

You should have no sites/default/settings.ddev.redis.php any more.

Make sure it doesn't break on DDEV v1.22.6 (which should ignore that extra section)

## Automated Testing Overview



## Related Issue Link(s)

## Release/Deployment Notes
